hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBStdSessions.java

```diff
@@ -120,13 +125,28 @@ public RocksDBStdSessions(HugeConfig config, String dataPath,
                      "Expect same size of cf-handles and cf-names");
 
         // Collect CF Handles
+        this.cfs = new HashMap<>();
         for (int i = 0; i < cfs.size(); i++) {
             this.cfs.put(cfs.get(i), cfhs.get(i));
         }
 
+        this.refCount = new AtomicInteger(1);
+
         ingestExternalFile();
     }
 
+    private RocksDBStdSessions(HugeConfig config, String database, String store,
+                               RocksDBStdSessions origin) {
+        super(config, database, store);
+
+        this.rocksdb = origin.rocksdb;
+        this.sstFileManager = origin.sstFileManager;
+        this.cfs = origin.cfs;
+        this.refCount = origin.refCount;
```

Comment:
this.refCount = origin.refCount.incrementAndGet();

Reply:
They have to share the address of refCount